cpu/arm7_common/arm_cpu.c

```diff
@@ -25,6 +25,13 @@
 #define STACK_MARKER    (0x77777777)
 #define REGISTER_CNT    (12)
 
+__attribute__((used,section(".usr_stack"))) uint8_t usr_stack[USR_STACKSIZE];
```

Comment:
```suggestion
__attribute__((used, section(".usr_stack"))) uint8_t usr_stack[USR_STACKSIZE];
```

With a space it would be more consistent. (Most of RIOT code does use the space after the comma in the list of attributes, and this is also more consistent with the coding convention.) Please squash this right in!

Reply:
Ok!